Pod/Classes/BONText.h

```diff
@@ -97,21 +97,21 @@ typedef NS_ENUM(NSUInteger, BONFigureSpacing) {
  *
  *  @return An @c NSAttributedString object that is the result of interposing separator’s attributed string between the attributed strings of the elements of the array. If the array has no elements, returns an @c NSAttributedString object representing an empty string.
  */
-+ (NSAttributedString *)joinTexts:(NSArray *)texts withSeparator:(BONText *)separator;
++ (NSAttributedString *_Nonnull)joinTexts:(NSArray<BONText *> *_Nullable)texts withSeparator:(BONText *_Nullable)separator;
 
 /**
  *  Calls [self debugStringIncludeImageAddresses:YES]
  *
  *  @return The debug string, including pointer addresses of attached images.
  */
-- (NSString *)debugString;
+- (NSString *_Nonnull)debugString;
```

Comment:
D'oh, so that's how you do it. I stopped at the first one I successfully got to compile. Fixing.